Solutions/Datalake2Sentinel/Data/Solution_Datalake2Sentinel.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Name": "Datalake2Sentinel",
+    "Author": "OrangeCyberdefense",
+    "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/OrangeCyberdefenseLogo.svg\" width=\"75px\" height=\"75px\">",
+    "Description": "The Datalake2Sentinel solution allows you to push threat indicators from Orange Cyberdefense Datalake platform to Microsoft Sentinel via the Upload Indicators REST API.\r\n\n",
```

Comment:
Also please add a few more lines to the description

Reply:
Ok